app/experimenter/experiments/tests/api/v5/test_mutations.py

```diff
@@ -71,7 +71,7 @@ def test_create_experiment(self):
             },
             headers={settings.OPENIDC_EMAIL_HEADER: user_email},
         )
-        self.assertEqual(response.status_code, 200)
+        self.assertEqual(response.status_code, 200, response.content)
```

Comment:
why response. content? aren't we just checking status code here

Reply:
The third argument gets printed out if the assertion fails, so it makes inspecting test failures easier.